### PR TITLE
Create show command

### PR DESCRIPTION
**Pull request by jclipa-lightit**
Create show command

ClickUp task [86du9b9vn](https://app.clickup.com/t/86du9b9vn)

### CODE REVIEW

File: movepedia [hunk 1]
```diff
@@ -1,9 +1,14 @@
 #! /usr/bin/env php
```
Comment (mateorodriguezcastillo) [inline movepedia:1]: Why is this here?

**Review by mateorodriguezcastillo — APPROVED** [review]

File: movepedia (hunk 1 above)
Comment (jclipa-lightit) [reply movepedia:1]: It is required for auto-detection of the type of script
